Content.Shared/_White/Wizard/SharedSpellsSystem.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System.Linq;
+using Content.Shared.Actions;
+using Content.Shared.Carrying;
+using Content.Shared.Charges.Components;
+using Content.Shared.Charges.Systems;
+using Content.Shared.Hands.Components;
+using Content.Shared.Hands.EntitySystems;
+using Content.Shared.IdentityManagement;
+using Content.Shared.Magic;
+using Content.Shared.Magic.Components;
+using Content.Shared.Movement.Pulling.Components;
+using Content.Shared.Popups;
+using Content.Shared.Tag;
+using Content.Shared.Weapons.Ranged.Components;
+using Content.Shared.Weapons.Ranged.Systems;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.Timing;
+
+
+namespace Content.Shared._White.Wizard;
+
```

Comment:
```suggestion
namespace Content.Shared._White.Spells;
```

Я бы перенес это из папки мага в папку спелов